src/Rings/mpoly-localizations.jl

```diff
@@ -1585,9 +1589,6 @@ function saturated_ideal(
     I.saturated_ideal = result
     if with_generator_transition
       error("computation of the transition matrix for the generators is not supposed to happen because of using local orderings")
-      for g in gens(result) 
-        g in I || error("generator not found") # assures caching with transitions
-      end
```

Comment:
Why remove this? This is important!

Reply:
This would be important if it were reached, but it was located after an Error-Exception and before the respective end, so there was no chance that it would be reached.